src/styles/wishList.css

```diff
@@ -31,7 +31,17 @@
 
 .btn-addToCart{
     margin: 0.625rem;
-    background-color: #FFA500;
+    background-color: #F7CA00;
```

Comment:
If the styles are being used in more than one place, consider defining it as global styles instead